src/components/ScreenWrapper/index.js

```diff
@@ -110,19 +111,21 @@ class ScreenWrapper extends React.Component {
                             ]}
                         >
                             <KeyboardAvoidingView style={[styles.w100, styles.h100, {maxHeight: this.props.windowHeight}]} behavior={this.props.keyboardAvoidingViewBehavior}>
-                                <HeaderGap />
-                                {// If props.children is a function, call it to provide the insets to the children.
-                                    _.isFunction(this.props.children)
-                                        ? this.props.children({
-                                            insets,
-                                            safeAreaPaddingBottomStyle,
-                                            didScreenTransitionEnd: this.state.didScreenTransitionEnd,
-                                        })
-                                        : this.props.children
-                                }
-                                {this.props.isSmallScreenWidth && (
-                                    <OfflineIndicator />
-                                )}
+                                <PickerAvoidingView>
```

Comment:
From the user's perspective, things looks like this:

https://user-images.githubusercontent.com/2590174/231263997-5966da44-6ec6-4610-837e-14b1de13fcbc.mp4

I'd like to confirm that this is the expected effect.

Reply:
@0xmiroslav Was this meant to be video? Because it's PNG, I'm not sure how that happened

Reply:
I'm sorry, I confused the behavior I remembered from iOS Safari / Web. I have a solution that seems acceptable. I'll push it in a few moments.

Reply:
@0xmiroslav @Julesssss 

This is how the solution looks with a subtle animation:

https://user-images.githubusercontent.com/2590174/231567854-2f5f7de5-6c5a-460c-94bc-0bdef46bf182.mp4

Reply:
I understand the problem now. I'm on it.

Reply:
Disabled:

https://user-images.githubusercontent.com/2590174/231866214-775d03d8-8c53-436e-bdb2-aba168b138b9.mp4

Enabled:

https://user-images.githubusercontent.com/2590174/231866227-f69df8e9-928d-437d-908d-00e13709eb8d.mp4